public/favicon.ico:Zone.Identifier

```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]

```

Comment:
I don't think this was meant to be committed

Reply:
The whole file sorry

Reply:
ok, thanks. Going to keep looking into the other 2 issues.